src/components/atomic/TextInput/TextInput.jsx

```diff
@@ -0,0 +1,158 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+
+import './TextInput.scss';
```

Comment:
should be pulled in via dictionaries.scss

Reply:
Done